### DISCUSSION TITLE
[R82XX] PLL not locked! (?)

### DISCUSSION THREAD

**Discussion by OE1MWW** [jvde-github/AIS-catcher#157] [General]
Hi all,
on my RTL-SDR V3 I have, as reported,  excelent sensivity
Nevertheless, on startup I see this "PLL not locked" twice.

What is the reason and consequences for such message ?

--- info at startup:

[R82XX] PLL not locked!
[R82XX] PLL not locked!
Device    : RTL2838UHIDIR
Settings  : rate 1536K bw 192K format CU8 tuner 38.000000 rtlagc ON biastee OFF buffer_count 24
Model #0 -> (Src: 1, Grp: 1): [AIS engine v0.50] ps_ema ON afc_wide ON droop ON fp_ds OFF dsk OFF
[AIS engine v0.50 #0-0]                 received: 27 msgs, total: 27 msgs, rate: 2.58214 msg/s

**Comment by jvde-github** [comment]
Hi, 

This is a message from the underlying RTL SDR library and there is no impact on decoding and reception. 

I don't know the technicalities why this is shown, this is the only thing I could find, but perhaps there are better references/explanations:

https://www.reddit.com/r/RTLSDR/comments/xxfplh/pll_not_locked/

Jasper

**Comment by jvde-github** [comment]
This message should now be gone. Stumbled upon the solution how to remove it.
